discounts/rust/order-discounts/volume-discount-i18n/README.md

```diff
@@ -0,0 +1,21 @@
+# Shopify Function development with Rust
```

Comment:
Hm- is this examples repository intended to be browsed? 🤔 

There's a lot of misc stuff in here, notably functions for unreleased APIs. I think there might be a fair amount of work involved in polishing this to be a good source of truth.

Reply:
I was under the impression that since it's public, and we have a set of examples here, that partners might take a look at it. If that's not the case, then ignore my comment 😄